app/src/main/AndroidManifest.xml

```diff
@@ -18,13 +18,10 @@
         android:theme="@style/Theme.PlaylistMaker"
         tools:targetApi="31">
 
-        <activity
-            android:name=".ui.audioplayer.AudioPlayerActivity"
-            android:exported="false"/>
         <activity
             android:name=".ui.root.RootActivity"
             android:exported="true"
-            android:windowSoftInputMode="adjustNothing">
+            android:windowSoftInputMode="adjustPan">
```

Comment:
лучше использовать adjustResize 
в чем приемущество - "adjustResize" - можно почитать в этом вопросе на stackoverflow.
Мне больше всего понравился этот ответ :

The activity's main window is always resized to make room for the soft keyboard on screen.

"adjustPan"

The activity's main window is not resized to make room for the soft keyboard. Rather, the contents of the window are automatically panned so that the current focus is never obscured by the keyboard and users can always see what they are typing. This is generally less desirable than resizing, because the user may need to close the soft keyboard to get at and interact with obscured parts of the window.